scripts/xlxd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=XLXD Multiprotocol Reflector
+After=network.target ambed.service
+
+[Service]
+Type=forking
```

Comment:
for running in systemd, it would make more sense to compile it to not use the deamon mode and say simple. Then you can even use stdout / stderr for logging if you want.